recipes/pybind11_json/all/conanfile.py

```diff
@@ -0,0 +1,48 @@
+from conans import ConanFile, tools
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class Pybind11JsonConan(ConanFile):
+    name = "pybind11_json"
+    homepage = "https://github.com/pybind/pybind11_json"
+    description = "An nlohmann_json to pybind11 bridge"
+    topics = (
+        "conan",
+        "header-only",
+        "json",
+        "nlohmann_json",
+        "pybind11",
+        "pybind11_json",
+        "python",
+        "python-binding",
+    )
+    url = "https://github.com/conan-io/conan-center-index"
+    no_copy_source = True
+    license = "BSD-3-Clause"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def package_id(self):
+        self.info.header_only()
+
+    def requirements(self):
+        self.requires("nlohmann_json/3.9.1")
+        self.requires("pybind11/2.6.2")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+                  destination=self._source_subfolder, strip_root=True)
+
+    def package(self):
+        self.copy(pattern="LICENSE*", dst="licenses", src=self._source_subfolder)
```

Comment:
```suggestion
        self.copy(pattern="LICENSE*", src=self._source_subfolder, dst="licenses")
```
OCD 😛

Reply:
Indeed, I don't get how people do `dst=..., src=...`.
Unix' `cp` and Windows' copy also has src first.

Reply:
Good catch, I should have scrutinized more of my copy-paste from `nlohmann_json` :sweat_smile:

I guess I'll do a follow-up PR fixing these minor issues in `nlohmann_json`.

Reply:
It's no bug to do `dst=` first.
It's just odd that it is documented this way.

Reply:
I know, but still, this oddity lives on and spreads (as demonstrated in this PR) if we don't purge it.

Reply:
Yes, kill it with :fire: